internal/guest/bridge/bridge_v2.go

```diff
@@ -133,16 +136,39 @@ func (b *Bridge) createContainerV2(r *Request) (_ RequestResponse, err error) {
 //
 // This is allowed only for protocol version 4+, schema version 2.1+
 func (b *Bridge) startContainerV2(r *Request) (_ RequestResponse, err error) {
-	_, span := trace.StartSpan(r.Context, "opengcs::bridge::startContainerV2")
+	ctx, span := trace.StartSpan(r.Context, "opengcs::bridge::startContainerV2")
 	defer span.End()
 	defer func() { oc.SetSpanStatus(span, err) }()
-	span.AddAttributes(trace.StringAttribute("cid", r.ContainerID))
+	span.AddAttributes(trace.StringAttribute(logfields.ContainerID, r.ContainerID))
+	entity := log.G(ctx).WithField(logfields.ContainerID, r.ContainerID)
 
 	// This is just a noop, but needs to be handled so that an error isn't
 	// returned to the HCS.
 	var request prot.MessageBase
-	if err := commonutils.UnmarshalJSONWithHresult(r.Message, &request); err != nil {
-		return nil, errors.Wrapf(err, "failed to unmarshal JSON in message \"%s\"", r.Message)
+	if err = commonutils.UnmarshalJSONWithHresult(r.Message, &request); err != nil {
```

Comment:
I'm fine keeping the named returns, but I prefer explicitly returning `nil, errors.Wrapf(err, "failed to unmarshal JSON in message \"%s\"", r.Message)` or something here

Reply:
updated